images/devcontainer/features.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Dev Container Features
+---
+
+## Overview
+
+The Rocker Project provides some [Dev Container Features](https://containers.dev/implementors/features/)
+for installing R or installing software often used with R.
+
+- Source repository: [rocker-org/devcontainer-features](https://github.com/rocker-org/devcontainer-features)
+
+You can find them on the [Dev Containers site](https://containers.dev/collections),
+[the GitHub Codespaces Dev Container Editor](https://github.blog/changelog/2022-10-21-codespaces-configuration-with-the-dev-container-editor/),
+or, [VSCode Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers).
+
+These can be used to easily configure R on containers without R installed,
+or to make containers for R even more useful.
+And of course, we can use them independently of R! (except for the Features to install R)
+
+For example, install R on the default image of Codespaces
+by editing [the `devcontainer.json` file](https://containers.dev/implementors/spec/#devcontainerjson) as follows:
+
+```{.json filename=".devcontainer/devcontainer.json"}
+{
+    "image": "mcr.microsoft.com/devcontainers/universal:2",
+    "features": {
+        "ghcr.io/rocker-org/devcontainer-features/r-apt:latest": {}
+    }
+}
+```
+
+Check the source repository for details on each Feature.
+
+## Install R on Dev Containers
+
+1. [`r-apt`](https://github.com/rocker-org/devcontainer-features/tree/main/src/r-apt)
+2. [`r-rig`](https://github.com/rocker-org/devcontainer-features/tree/main/src/r-rig)
+3. [`miniforge`](https://github.com/rocker-org/devcontainer-features/tree/main/src/miniforge)[^miniforge]
+
+[^miniforge]: This Feature does not directly install R, but it configures mamba so we can use mamba to install R.
+
+Each of these installations of R will be configured to allow installation of R binary packages in the following ways.
+
+1. [System package management system (`apt`)](../../use/extending.md#system-package-management-system)
+2. [RStudio Public Package Manager (R functions e.g. `install.packages`)](../../use/extending.md#rstudio-public-package-manager)[^rspm]
```

Comment:
AFAIK it is now called Posit Package Manager (PPM).  I probably will keep RSPM around in my documentation too for a bit.

Reply:
For now the home page of the website seems to be RStudio Package Manager, so I have left it as it is.
It can be updated later.

Reply:
It's a Heisenproduct :)  It has both names and links:  https://docs.posit.co/rpm/

Reply:
I see. https://posit.co/products/cloud/public-package-manager/